docs/source/user/getting_started/advanced_workflow_concepts.rst

```diff
@@ -0,0 +1,308 @@
+        scheduler_args:
+        shebang_args: --login
+        options:
+            -l: short
+
+
+or at the task level
+
+.. code-block:: yaml
+
+    - schema: simulate_VE_loading_damask
+    resources:
+        any:
+        # This will use two cores for input file generators and output file parsers
+        num_cores: 2
+        main:
+        # Use 16 cores for the "main" part of the task (the simulation in this case)
+        num_cores: 16
+    inputs:
+        ...
+
+
+We can see from above that it is possible to request resources for subsets of the actions
+in a task schema. A full list of the different options you can select resources for is given below.
+
+- ``input_file_generator``
+- ``output_file_parser``
+- ``processing`` (a shortcut for ``input_file_generator`` +  ``output_file_parser``)
+- ``main`` (the main part of the action i.e. not ``processing``)
+- ``any`` (anything not already specified with any of the above options)
+
+These are used to choose resources (done at the workflow/task level),
+and also the same values can be used within the schema to select an ``environment``
+by ``scope`` e.g.
+
+.. code-block:: yaml
+
+    actions:
+    - environments:
+    - scope:
+        type: processing
+        environment: damask_parse_env
+    - scope:
+        type: main
+        environment: damask_env
+
+
+MatFlow is then looking for a match within your environment definitions for the requested
+resources, and will run the command which matches those resources.
+
+There are lots of :ref:`resource options <reference/_autosummary/matflow.ResourceSpec:matflow.ResourceSpec>`
+available that can be requested.
+
+Scheduler arguments can be passed like this e.g. to target high memory nodes:
+
+.. code-block:: yaml
+
+    resources:
+    any:
+        num_cores: 10
+        SGE_parallel_env: smp.pe
+        scheduler_args:
+        options:
+            -l: mem512
+
+
+If you have set resource options at the top level (for the whole workflow), but would like to "unset" them for a particular task,
+
+you can pass an empty dictionary:
+
+.. code-block:: yaml
+
+    - schema: simulate_VE_loading_damask
+    resources:
+        main:
+        num_cores: 16
+        scheduler_args:
+            options: {} # "Clear" any previous options which have been set.
+    inputs:
+
+
+Task sequences
+----------------
+
+Matflow can run tasks over a set of inputs values, if the iterations are independent.
+For this, you use a ``sequence``, and a ``nesting_order`` to control the nesting of the loops
+but you can also "zip" two or more lists of inputs by using the same level of nesting.
+Lower values of ``nesting_order`` act like the "outer" loop.
+
+.. code-block:: yaml
+
+    tasks:
+    - schema: my_schema
+    sequences:
+    - path: inputs.conductance_value
+    values:
+    - 0
+    - 100
+    - 200
+    nesting_order: 0
+
+Groups
+-------
+
+To combine outputs from multiple sequences, you can use a ``group`` in a task schema:
```

Comment:
Groups are for combining multiple elements. Those elements may have come from a single sequence, or multiple sequences, or from a `repeats` parameter.

Reply:
done